src/main/java/uk/gov/hmcts/ccd/domain/service/common/CaseAccessService.java

```diff
@@ -68,6 +72,10 @@ public AccessLevel getAccessLevel(ServiceAndUserDetails serviceAndUserDetails) {
         return new HashSet<>(caseUserRepository.findCaseRoles(Long.valueOf(caseId), userRepository.getUserId()));
     }
 
+    public Set<String> getCaseCreationCaseRoles() {
+        return Sets.union(getUserRoles(), singleton(GlobalCaseRole.CREATOR.getRole()));
```

Comment:
`singleton(GlobalCaseRole.CREATOR.getRole())` is duplicated. 
We could introduce a method 
`getCaseCreationCaseRoles()` that will be called in `AuthorisedStartEventOperation.getCaseRoles`. Here it would be:
```
public Set<String> getCaseCreationRoles() {
return Sets.union(getUserRoles(), getCaseCreationCaseRoles();
}
```